src/java.base/share/classes/java/lang/module/Resolver.java

```diff
@@ -498,12 +498,11 @@ private void checkHashes() {
      */
     private Map<ResolvedModule, Set<ResolvedModule>> makeGraph(Configuration cf) {
 
-        // initial capacity of maps to avoid resizing
-        int capacity = 1 + (4 * nameToReference.size())/ 3;
+        int nameToReferenceSize = nameToReference.size();
```

Comment:
This can be renamed moduleCount to make it easier to read.

Reply:
@AlanBateman done.